src/app/colors/colors.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { ColorAndTooltip } from './colors.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ColorsService {
```

Comment:
an entire service for this seems a bit a waste of memory and effort

I strongly suggest you to put this value in a file.model.ts or similar (ex. .data.ts), using a Service block a bit the tree-shaking and the web-packetization sometimes.